### PR TITLE
guides: replace pep517 CLI usage with pypa/build

### PR DESCRIPTION
**Pull request by FFY00**
See https://github.com/pypa/pep517/issues/91

Signed-off-by: Filipe Laíns <lains@riseup.net>

### CODE REVIEW

File: source/guides/github-actions-ci-cd-sample/publish-to-test-pypi.yml [hunk 1]
```diff
@@ -13,19 +13,19 @@ jobs:
...
     - name: Build a binary wheel and a source tarball
       run: >-
         python -m
-        pep517.build
-        --source
-        --binary
-        --out-dir dist/
+        build
+        --sdist
+        --weel
```
Comment (pfmoore) [inline source/guides/github-actions-ci-cd-sample/publish-to-test-pypi.yml:27]: Typo, I assume?

Comment (FFY00) [reply source/guides/github-actions-ci-cd-sample/publish-to-test-pypi.yml:27]: Yes, ugh, sorry. I will fix it up when I get to the computer.

**Review by pfmoore — APPROVED** [review]

File: source/guides/github-actions-ci-cd-sample/publish-to-test-pypi.yml [hunk 2]
```diff
@@ -13,19 +13,19 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: 3.7
-    - name: Install pep517
+    - name: Install build
```
Comment (webknjaz) [inline source/guides/github-actions-ci-cd-sample/publish-to-test-pypi.yml:16]: how about
```suggestion
    - name: Install python-build
```

**Review by webknjaz — APPROVED** [review]
LGTM overall

File: source/guides/github-actions-ci-cd-sample/publish-to-test-pypi.yml (hunk 2 above)
Comment (FFY00) [reply source/guides/github-actions-ci-cd-sample/publish-to-test-pypi.yml:16]: I think `pypa/build` would be better in that case.

Comment (webknjaz) [reply source/guides/github-actions-ci-cd-sample/publish-to-test-pypi.yml:16]: makes sense

File: source/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows.rst [hunk 3]
```diff
@@ -87,13 +87,13 @@ Then, add the following under the ``build-n-publish`` section:
 .. literalinclude:: github-actions-ci-cd-sample/publish-to-test-pypi.yml
    :language: yaml
    :start-after: runs-on:
-   :end-before: Install pep517
+   :end-before: Install build
```
Comment (webknjaz) [inline source/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows.rst:90]: The marker needs to match the change above 
```suggestion
   :end-before: Install pypa/build
```

**Comment by pradyunsg** [comment]
Great! Thanks! ^>^

**Pull request merged**